hera_cal/apply_cal.py

```diff
@@ -206,10 +214,21 @@ def apply_cal(data_infilename, data_outfilename, new_calibration, old_calibratio
         add_to_history: appends a string to the history of the output file. This will preceed combined histories
             of flag_file (if applicable), new_calibration and, old_calibration (if applicable).
         clobber: if True, overwrites existing file at outfilename
+        redundant_average : bool, optional
+            If True, redundantly average calibrated data and save to <data_outfilename>.red_avg.<filetype_out>
+        redundant_weights : datacontainer, optional.
+            Datacontainer containing weights to use in redundant averaging.
+            only used if redundant_average is True.
+            Default is None.
```

Comment:
Explain that if the default is None, then the nsamples is used.

Reply:
added this clarification.